test/elastic_record/relation_test.rb

```diff
@@ -28,12 +28,14 @@ def test_to_hits
   end
 
   def test_to_ids
+    Widget.elastic_index.delete_all
     create_widgets [Widget.new(id: 5, color: 'red'), Widget.new(id: 10, color: 'blue')]
 
     assert_equal ['5', '10'].to_set, Widget.elastic_relation.to_ids.to_set
   end
 
   def test_to_a
+    Widget.elastic_index.delete_all
```

Comment:
We do not have these failures in our apps, which do more complex things than here. I believe that is enough evidence to show that something is going wrong here.

This gem was originally made without a test/dummy app. I am thinking of changing the test suite to use real active_record models found in test/dummy/app/models.